controllers/common/common.go

```diff
@@ -60,24 +62,25 @@ func IsReplicatedPolicy(c client.Client, policy client.Object) (bool, error) {
 	return IsInClusterNamespace(c, policy.GetNamespace())
 }
 
-// IsPbForPoicy compares group and kind with policy group and kind for given pb
-func IsPbForPoicy(pb *policiesv1.PlacementBinding) bool {
-	found := false
-
-	subjects := pb.Subjects
-	for _, subject := range subjects {
-		if subject.Kind == policiesv1.Kind && subject.APIGroup == policiesv1.SchemeGroupVersion.Group {
-			found = true
+// IsForPolicyOrPolicySet returns true if any of the subjects of the PlacementBinding are Policies
+// or PolicySets.
+func IsForPolicyOrPolicySet(pb *policiesv1.PlacementBinding) bool {
+	if pb == nil {
+		return false
+	}
 
-			break
+	for _, subject := range pb.Subjects {
+		if subject.APIGroup == policiesv1.SchemeGroupVersion.Group &&
+			(subject.Kind == policiesv1.Kind || subject.Kind == policiesv1.PolicySetKind) {
+			return true
```

Comment:
I prefer adding `break`, if we find policy... why did you remove this

Reply:
Why is `break` better? I feel like it's the same

Reply:
oh yeah same!

Reply:
I was working with javascript so confused lol